net.fsuae.FS-UAE.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "net.fsuae.FS-UAE",
+    "branch": "stable",
+    "command": "fs-uae",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "rename-desktop-file": "fs-uae.desktop",
+    "rename-icon": "fs-uae",
+    "finish-args": [
+        "--device=all",
+        "--filesystem=home",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=x11"
+    ],
+    "modules": [
+        {
+            "name": "libmpeg2",
+            "cleanup": [ "/bin", "/include", "/lib/pkgconfig", "/lib/*.la" ],
+            "config-opts": [ "--disable-static" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz",
+                    "sha256": "dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4"
+                }
+            ]
+        },
+        {
+            "name": "fs-uae",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap",
+                "./configure",
+                "make",
+                "make prefix=/app install"
+            ],
+            "cleanup": [
+                "/share/man",
+                "/share/doc"
+            ],
+            "post-install": [
+                "install -Dm644 net.fsuae.FS-UAE.appdata.xml /app/share/appdata/net.fsuae.FS-UAE.appdata.xml",
+                "sed -i s/NoDisplay=.*/NoDisplay=false/ /app/share/applications/fs-uae.desktop",
+                "cp /app/share/mime/packages/fs-uae.xml /app/share/mime/packages/net.fsuae.FS-UAE.xml"
+            ],            
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/FrodeSolheim/fs-uae",
+                    "branch": "stable"
```

Comment:
You should add specific `commit` here so the build is deterministic.

Reply:
Done